src/app/tests/TestReadInteraction.cpp

```diff
@@ -1840,13 +1838,15 @@ void TestReadInteraction::TestSubscribeWildcard(nlTestSuite * apSuite, void * ap
             err = engine->GetReportingEngine().SetDirty(dirtyPath);
             NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
-            ctx.DrainAndServiceIO();
-
             //
-            // Not sure why I had to add this, and didn't have cycles to figure out why.
```

Comment:
Please leave the comments explaining _what_ is being tracked (modified to the new setup as needed).